api/bazel/BUILD

```diff
@@ -10,3 +11,10 @@ go_proto_compiler(
     valid_archive = False,
     visibility = ["//visibility:public"],
 )
+
+go_binary(
```

Comment:
not sure if this is correct/necessary - i figured we need to compile the lib/bin somewhere - locally i did it with `make build` and then exporting `./bin` to `PATH`